libs/ledger-live-common/src/env.ts

```diff
@@ -152,13 +137,8 @@ const envDefinitions = {
     parser: boolParser,
     desc: "Static fee for Stellar account",
   },
-  API_OSMOSIS_INDEXER: {
-    def: "https://osmosis.coin.ledger.com/indexer",
-    parser: stringParser,
-    desc: "Endpoint for Transaction Explorer/Indexer for Osmosis",
-  },
   API_OSMOSIS_NODE: {
-    def: "https://osmosis.coin.ledger.com/node",
+    def: "https://osmosis.coin.ledger-stg.com",
```

Comment:
should be production one no ?

Reply:
it is for temporary test. We will change it to prod after QA validation